libr/core/cmd_open.c

```diff
@@ -888,6 +886,13 @@ R_API void r_core_file_reopen_debug(RCore *core, const char *args) {
 		(void)r_core_cmd0 (core, "oo");
 		return;
 	}
+	bool is_file_uri = r_str_startswith (binpath, "file://");
+	if (is_file_uri) {
+		eprintf ("Trimming down the 'file://' URI handler.\n");
```

Comment:
Is it really useful to add a warning message? A comment is enough imho, a developer might care while the end user probably not.

Reply:
uhm yeah better to remove this